docs/flow/modin/core/execution/ray/implementations/pandas_on_ray/index.rst

```diff
@@ -1,5 +1,30 @@
+PandasOnRay Execution
+=====================
+
+Flows which a user’s queries for data transformation, data ingress and egress pass through the Modin components
```

Comment:
We are trying to get rid of `backend` word. Since the recent restructure we use `execution` for describing the whole name of a component that consists of a storage format and an engine. That might also be `execution system` or `execution backend`.